source/content/drupal-9.md

```diff
@@ -95,6 +95,49 @@ $node = Node::load(1);
 
 Since most of these changes are relatively minor, there are a number of [deprecation checking and correction tools](https://www.drupal.org/docs/9/how-to-prepare-your-drupal-7-or-8-site-for-drupal-9/deprecation-checking-and-correction-tools) available.
 
+## Test-upgrade an Existing Pantheon Drupal 8 Site to Drupal 9
+
+Are you already running a Pantheon site using our [Drupal 8 upstream](https://github.com/pantheon-systems/drops-8)? Leverage the amazing power of our Multidev tooling to test Drupal 9 in a Git branch.
+
+1. Clone your Drupal 8 [site’s codebase to your computer](/local-development#get-the-code). You can create a new Drupal 8 site or use an existing Drupal 8 site:
+
+  ```bash{promptUser: user}
+  git clone <url for site repo>
+  ```
+
+1. Use Terminus to create a [Multidev](/multidev) environment called `d9-preview` on your Drupal 8 site for testing:
+
+  ```bash{promptUser: user}
+  terminus multidev:create <site>.<env> d9-preview
+  ```
+
+1. Create and switch to a new testing branch in the Drupal 8 codebase:
+
+  ```bash{promptUser: user}
+  git checkout -b d9-preview
+  ```
+
+1. Pull the `drupal9.0.0` tag into your codebase:
+
+  ```bash{promptUser: user}
+  git pull https://github.com/pantheon-systems/drops-8.git drupal9.0.0--from-8.9.0--early-access-not-for-production-use
```

Comment:
`drupal9.0.0--from-8.9.0--early-access-not-for-production-use` should just be `9.0.0`